client/my-sites/customer-home/free-photo-library-card/index.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { localize } from 'i18n-calypso';
+import { Card, Button, Dialog } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import { localizeUrl } from 'lib/i18n-utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Image dependencies
+ */
+import freePhotoLibraryThumbnail from 'assets/images/customer-home/free-photo-library-thumbnail.png';
+import freePhotoLibraryVideoPrompt from 'assets/images/customer-home/free-photo-library-video-prompt.png';
+
+class FreePhotoLibraryCard extends Component {
+	state = {
+		showDialog: false,
+	};
+
+	toggleDialog = () => {
+		this.setState( state => ( {
+			showDialog: ! state.showDialog,
+		} ) );
+	};
+
+	render() {
+		const { translate } = this.props;
+		return (
+			<Fragment>
+				<Dialog
+					additionalClassNames="free-photo-library-card__dialog"
+					isVisible={ this.state.showDialog }
+				>
+					<video
+						className="free-photo-library-card__demonstration-video"
+						controls
+						muted
+						poster={ freePhotoLibraryThumbnail }
+					>
+						<source
+							src="https://wpcom.files.wordpress.com/2020/02/free-photo-library-demonstration.mp4"
+							type="video/mp4"
+						/>
+					</video>
+					<div className="free-photo-library-card__dialog-close">
+						<Button onClick={ this.toggleDialog }>{ translate( 'Close' ) }</Button>
+					</div>
+				</Dialog>
+				<Card className="free-photo-library-card">
+					{ /* eslint-disable */ }
```

Comment:
Which lint error were we running into? We can help debug the issue. 

If we really need to disable, let's be specific about which rule we're turning off. We'll miss other nice lint checks for the codeblock otherwise.

For example:
```
/* eslint-disable jsx-a11y/anchor-is-valid */
...some code
/* eslint-enable jsx-a11y/anchor-is-valid */
```

Reply:
It's the `no-noninteractive-element-interactions `and the `no-noninteractive-tabindex` ones. Is there a way to fix them or should I just disable those two rules specifically?

Reply:
* `no-noninteractive-element-interactions`: rather than adding an `onClick` to a `img` which is non-interactive, you can wrap it in a button or anchor.
*  `no-noninteractive-tabindex`: this should be fixable by removing the `tabIndex` param. Did you add it for any specific reason?

Reply:
We have some [instances of disabling `no-noninteractive-element-interactions`](https://github.com/Automattic/wp-calypso/search?q=no-noninteractive-element-interactions&unscoped_q=no-noninteractive-element-interactions) though, so feel free to disable the rule specifically. But from an accessibility point of view I think it's better to have an interactive element like a button or a link since that provides extra context on screen readers.

Reply:
I was using `tabIndex` to try and get the image to focus when navigating with just a keyboard, but I wrapped the image in a button which makes the linter happy now. Thanks @mmtr! :)